hack/examples/patches/nutanix/csi.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+- op: "add"
+  path: "/spec/topology/variables/0/value/addons/csi"
+  value:
+    defaultStorage:
+      providerName: nutanix
+      storageClassConfigName: nutanix-volume
```

Comment:
```
The Cluster "dkoshkin-nutanix" is invalid: spec.topology.variables.addons.csi.defaultStorage.storageClassConfigName: Required value
```

We do need this because of the API. I will put it back in this PR and then we can do a follow up to either change the API or remove the special handling in the CSI driver with a single SC